pynopegl-utils/pynopegl_utils/export.py

```diff
@@ -199,3 +199,72 @@ def _export_worker(
 
     os.close(fd_w)
     reader.wait()
+
+
+def run():
+    import argparse
+    from inspect import getmembers
+    from pathlib import Path
+
+    from pynopegl_utils.misc import SceneCfg
+    from pynopegl_utils.module import load_script
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        dest="input",
+        type=Path,
+        help="path to the script containing scenes",
+    )
+    parser.add_argument(
+        "-o",
+        dest="output",
+        type=Path,
+        default="ngl-exports",
+        help="path to the export directory",
+    )
+    parser.add_argument(
+        "-r",
+        dest="resolution",
+        default="1080p",
+        choices=RESOLUTIONS.keys(),
+        help="output video resolution",
+    )
+    parser.add_argument(
+        "-p",
+        dest="profile",
+        default="nut_ffv1",
+        choices=ENCODE_PROFILES.keys(),
+        help="output video profile",
+    )
+    parser.add_argument(
+        "-m",
+        dest="samples",
+        type=int,
+        default=1,
+        choices=[0, 1, 2, 4, 8],
+        help="number of samples used for multisample anti-aliasing",
+    )
```

Comment:
can you add the ability to specify a scene name (could be a list ?) so not all the scenes are exported but the specified ones ? This would make the tool more flexible and also useful when you want to debug a specific scene.